src/test/java/com/conveyal/gtfs/GTFSTest.java

```diff
@@ -424,25 +604,10 @@ private boolean runIntegrationTestOnZipFile(
             throw e;
         }
 
-        // Verify that making a snapshot from an existing feed database, then exporting that snapshot to a GTFS zip file
-        // works as expected
-        boolean snapshotIsOk = assertThatSnapshotIsSuccessful(namespace, dataSource, testDBName, persistenceExpectations, false);
-        if (!snapshotIsOk) return false;
-        // Also, verify that if we're normalizing stop_times#stop_sequence, the stop_sequence values conform with our
-        // expectations (zero-based, incrementing values).
-        PersistenceExpectation[] expectationsWithNormalizedStopTimesSequence =
-            updatePersistenceExpectationsWithNormalizedStopTimesSequence(persistenceExpectations);
-        boolean normalizedSnapshotIsOk = assertThatSnapshotIsSuccessful(
-            namespace,
-            dataSource,
-            testDBName,
-            expectationsWithNormalizedStopTimesSequence,
-            true
-        );
-        if (!normalizedSnapshotIsOk) return false;
-
-        // Verify that deleting a feed works as expected.
+        // Get a new connection here, because sometimes the old connection causes hanging issues upon trying to drop a
+        // schema (via deleting a GTFS namespace).
```

Comment:
Which old connection are you referencing here? Nothing was changed in the code related to getting a new connection as far as I can tell.